src/pages/components/pagination.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Pagination
+description: >
+  Pagination separates content into pages and allows users to navigate between those pages.
+package: '@zendeskgarden/react-pagination'
+components:
+  - pagination/src/elements/Pagination/Pagination.tsx
+  - pagination/src/elements/CursorPagination/CursorPagination.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To divide large amounts of data into manageable chunks</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Offset pagination
+
+The basic usage of an offset pagination component.
+
+import OffsetDefault from '../../examples/pagination/OffsetDefault.tsx';
+import OffsetDefaultCode from '!!raw-loader!../../examples/pagination/OffsetDefault.tsx';
+
+<CodeExample code={OffsetDefaultCode}>
+  <OffsetDefault />
+</CodeExample>
+
+### Gap indicator
+
+The position of a gap indicator (...) can be set in the sequence to represent overflowing pages.
+
+import OffsetGap from '../../examples/pagination/OffsetGap.tsx';
+import OffsetGapCode from '!!raw-loader!../../examples/pagination/OffsetGap.tsx';
+
+<CodeExample code={OffsetGapCode}>
+  <OffsetGap />
+</CodeExample>
+
+### Page padding
+
+The amount of page numbers to display (not including page gap indicators or
+arrows) is modified with `pagePadding`. Each increment adds 2 page numbers.
+
+import OffsetPadding from '../../examples/pagination/OffsetPadding.tsx';
+import OffsetPaddingCode from '!!raw-loader!../../examples/pagination/OffsetPadding.tsx';
+
+<CodeExample code={OffsetPaddingCode}>
+  <OffsetPadding />
+</CodeExample>
+
+### Cursor pagination
```

Comment:
I believe it's "Cursor-based pagination" but check with @vsrixyz for whether or not the dash is correct.